route-rs-runtime/Cargo.toml

```diff
@@ -8,4 +8,5 @@ license = "MIT"
 [dependencies]
 tokio = "*"
 futures = "*"
-crossbeam = "*"
\ No newline at end of file
+crossbeam = "*"
+smoltcp = "*"
```

Comment:
We should probably pin the version of `smoltcp`.

Reply:
Good idea, especially since we are only using this for its internal structures.  Once async/await lands and we port tokio I plan on pinning all our dependencies, and maybe make a monthly issue to try and bump the versions when we can.